libs/pavexc_attr_parser/src/lib.rs

```diff
@@ -54,6 +54,11 @@ pub fn parse(
                     .map_err(InvalidAttributeParams::wrap)?;
                 parsed.into()
             }
+            "error_observer" => {
+                let parsed = model::ErrorObserverProperties::from_meta(&attr.meta)
+                    .map_err(InvalidAttributeParams::wrap)?;
```

Comment:
You are using the `InvalidAttributeParams::wrap` function here for `post_process`, `pre_process`, and `error_observer`. If this is intended, you might want to rename it.

Reply:
Copy-pasting left-overs, as it happens. Cleaned it up properly in [2df7bb6](https://github.com/LukeMathWalker/pavex/pull/484/commits/2df7bb60e5bdde4700d26aa7e6d60fd4ef9d94d5)